malaria24/ona/tasks.py

```diff
@@ -140,6 +144,16 @@ def make_pdf(html_content):  # pragma: no cover
     return value
 
 
+@celery_app.task(ignore_result=True)
+def compile_and_send_jembi(case_pk):
+    case = ReportedCase.objects.get(pk=case_pk)
+    api_url = 'http://jembi.org/malaria24'
```

Comment:
This api_url should also come from settings, if they decide to change the url we only have to update a env var not code.